h5netcdf/core.py

```diff
@@ -621,11 +621,18 @@ def _create_child_variable(
         if shape != maxshape:
             kwargs["maxshape"] = maxshape
 
+        # keep track of chunks
```

Comment:
This does not really help us. We can't overload the `chunks` to represent all possible situations:

- use h5py chunking, chunks=None, -> chunk only if maxshape is set
- use h5py chunking, chunks=True, -> chunk in any case
- use h5netcdf chunking, chunks=None, -> chunk only if maxshape is set
- use h5netcdf chunking, chunks=True, -> chunk in any case

Do you have a suggestion @dionhaefner?

Reply:
If we would add YAKWA `chunk_heuristics`, we could leave `chunks` as is and use `chunk_heuristics="h5netcdf"` to use h5netcdf-chunking and `chunk_heuristics="h5py"` for h5py-chunking. 

I do not see any other meaningful solution atm, so any input would be appreciated.